tracing-subscriber/src/filter/env/directive.rs

```diff
@@ -857,4 +859,16 @@ mod test {
         let dirs = parse_directives(format!("target[{}]=info", invalid_span_name));
         assert_eq!(dirs.len(), 0, "\nparsed: {:#?}", dirs);
     }
+
+    #[test]
+    fn parse_directives_with_whitespace() {
+        let dirs = parse_directives("target1=info,\ntarget2=debug,  target3=warn");
```

Comment:
it could be worth including some more complex filter directives, such as those on span fields, which can also include whitespace. this way, we would make sure those directives are still parsed correctly even when directives are also whitespace separated.